code/items/weapons/NewtonLauncher.cs

```diff
@@ -5,7 +5,7 @@
 namespace TTTReborn.Items
 {
     [Library("weapon_newtonlauncher")]
-    [Weapon(SlotType = SlotType.OffensiveEquipment, AmmoType = "pistol")]
+    [Weapon(SlotType = SlotType.OffensiveEquipment, AmmoType = "ammo_pistol")]
```

Comment:
can we use a type in that place? Would make much more sense imo.

Reply:
I agree, can we do this in a different PR? This PR is getting a bit large. I can handle the changes. (you are suggesting using an enum right?)

Reply:
https://github.com/TTTReborn/tttreborn/pull/347